### ISSUE TITLE
Take off end

### ISSUE THREAD

**Issue by QuentinG50** [Estia-advanced-programming/pandora-public#32]
The take off period do end when the pitch value became negative or when it stop raising ?

**Comment by WilliamDelamare** [comment]
For this project, the `pitch` values are not really reliable (very noisy). 
I suggested to build on the `yaw` values (cf wiki). Yaw values indicate when the aircraft starts going straight (plateaus to find), aka when the jet starts its cruise. Since the jet can perform turns during its flight, we define:
- take off: phase before the starts of the first plateau
- landing: phase after the end of the last plateau
- cruise: everything in between

**Comment by QuentinG50** [comment]
but we also saw big problems in th yaw values (files 201 line 26...)

**Comment by WilliamDelamare** [comment]
Yeah: these are real generated data. It goes from -1 (unknown) to 143.5 (sensor recorded). The ACMI files from where we took the data are supposed to work that way. 

Once again, a real phase detection algorithm should consider altitude, speed, engine power, pitch+roll+yaw etc..., and use more advanced mathematical tools. Here, we want to keep it simple.

Bonus: (**after** implementing the simple solution)
if you have a better solution, share it with us. We will then implement it to see how it corresponds to the real recorded flight (we can visualize them in another software we developed in python)
